### PR TITLE
Adding multi panel support to plot_bands

### PR DESCRIPTION
**Pull request by lwasser**
addresses https://github.com/earthlab/earthpy/issues/314

### CODE REVIEW

**Comment by lwasser** [comment]
@mbjoseph  this puppy is finally ready for review. cleaned up and all tests passing!!

File: earthpy/plot.py [hunk 1]
```diff
@@ -112,11 +168,8 @@ def plot_bands(
 
     Returns
     ----------
-    tuple
-        fig : figure object
-            The figure of the plotted band(s).
-        ax or axs : axes object(s)
-            The axes object(s) associated with the plot.
+    ax or axs : axes object(s)
+        The axes object(s) associated with the plot.
```
Comment (mbjoseph) [inline earthpy/plot.py]: Does it make sense to add some specificity here by saying what class the returned axes are? e.g., does this return an array of matplotlib.axes._subplots.AxesSubplot objects, or something else?

File: earthpy/plot.py [hunk 2]
```diff
@@ -71,6 +71,61 @@ def colorbar(mapobj, size="3%", pad=0.09):
     return fig.colorbar(mapobj, cax=cax)
 
 
+def _plot_image(
```
Comment (mbjoseph) [inline earthpy/plot.py]: Nice work refactoring this out into its own function!

Comment (lwasser) [reply earthpy/plot.py]: thank you @mbjoseph !! i had to think about it for a while and then finally realized how to implement it more efficiently! it actually makes managing the entire main function easier i just didn't know how to do it at first as my brain was making it more complicated than it needed to be!!

File: earthpy/plot.py (hunk 1 above)
Comment (lwasser) [reply earthpy/plot.py]: totally!

**Comment by lwasser** [comment]
ok @mbjoseph i've made that change to the docstring to clarify!! please let me know if you catch anything else with this PR!

**Comment by lwasser** [comment]
@mbjoseph this is failing one of the appveyor tests... it's because of a download that is not related to this PR and i suspect rebuilding the PR would fix it (i've been doing that with travis sometimes) but i don't see how i can reset a build in appveyor. thoughts on what to do here? review / merge or try to rebuild it i just don't know how.

**Pull request merged**